client/aws/errors.go

```diff
@@ -11,8 +11,23 @@ const (
 	ELBConfigurationMismatch = "already exists and it is configured with different parameters"
 )
 
-var (
-	malformedAmazonAccountIDError   = errgo.New("amazon account ID can only contain numbers")
-	wrongAmazonAccountIDLengthError = errgo.New("amazon account ID has the wrong size")
-	emptyAmazonAccountIDError       = errgo.New("amazon account ID cannot be empty")
-)
+var malformedAmazonAccountIDError = errgo.New("malformed amazon account ID, it should only contain numbers")
```

Comment:
The error message does not align with the variable name like with the other errors.